app/profile/page.tsx

```
@@ -0,0 +1,60 @@
+          <h1 className="text-2xl md:text-3xl font-bold" id="profile-heading">Your Profile</h1>
+          <p className="text-muted-foreground text-sm md:text-base" id="profile-description">
+            Manage your account settings and preferences
+          </p>
+        </div>
+        
+        <div className="bg-card rounded-lg border border-border p-4 sm:p-6">
+          <UserProfile
+            appearance={{
+              elements: {
+                rootBox: "w-full",
+                card: "shadow-none w-full p-0",
```

Biome (1.9.4)

[error] This property value named card is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named card


(lint/suspicious/noDuplicateObjectKeys)